webpack.config.js

```diff
@@ -1,14 +1,19 @@
-var path = require('path');
+const path = require('path');
+const webpack = require('webpack');
 
 module.exports = {
   entry: './src/api/v4/index.js',
   output: {
     path: path.resolve(__dirname, 'dist/public'),
-    filename: 'carto.bundle.js',
+    filename: 'carto.js',
     library: 'carto',
     libraryTarget: 'umd'
   },
-  // Tell webpack not to include Leaflet in the bundle. Leaflet should will be included manually
+  plugins: [
+    // Include only the lastest camshaft-reference
+    new webpack.IgnorePlugin(/^\.\/((?!0\.59\.4).)*\/reference\.json$/)
```

Comment:
how this works?

Reply:
This regex filter the `./0.59.3/reference.json` and previous versions, leaving only the latest version, that is the 0.59.4

Reply:
so if someone updates the required camshaft reference should update this file too right?

Reply:
Yes, and also the browserify file. This will be nices when we get rid of camshaft-reference.